### PR TITLE
Add HOS 8 manual fix warning about osconfig ansible (soc-11184)

### PR DESCRIPTION
**Pull request by jeremy-moffitt**
soc-11184_hos_warning

### CODE REVIEW

**Review by KeithMnemonic — DISMISSED** [review]
LGTM but Fergal should verify since he worked on this

File: xml/operations-maintenance-upgrade_maintenance.xml [hunk 1]
```diff
@@ -1084,6 +1084,40 @@ There are some running programs that might use files deleted by recent upgrade.
      The deployer node is now ready to be used to upgrade the remaining
      cloud nodes and running services.
    </para>
+   <warning>
+     <para>
+      If upgrading from Helion OpenStack 8, there is a manual file update that must
+      be applied before continuing the upgrade process. In the file
+      <literal>/usr/share/ardana/ansible/roles/osconfig/tasks/check-product-status.yml</literal>
+      replace `command` with `shell` in the first ansible entry. The correct version
+      of the file appears below.
```
Comment (rtamalin) [inline xml/operations-maintenance-upgrade_maintenance.xml:1093]: We probably should include the blurb about git commit'ing the change, after making it, as well ;-) Though if you try and run ready-deployment.yml as the next step with will complain anyway.

**Review by rtamalin — CHANGES_REQUESTED** [review]
So while the change in principle if good, to avoid people being silly, we probably should add the standard verbiage about needing to git commit this playbook change after making it.

File: xml/operations-maintenance-upgrade_maintenance.xml (hunk 1 above)
Comment (jeremy-moffitt) [reply xml/operations-maintenance-upgrade_maintenance.xml:1093]: thats good feedback, I'll fix it and upload another version, thanks!

**Comment by jeremy-moffitt** [comment]
soc-11184_hos_warning
updated version

**Review by rtamalin — DISMISSED** [review]
LGTM

**Comment by jeremy-moffitt** [comment]
had to adjust the commit message capitalization to hopefully pass the travisCI test

**Review by rtamalin — APPROVED** [review]
LGTM

**Pull request merged**